lib/ex_doc/formatter/html/autolink.ex

```diff
@@ -444,6 +446,10 @@ defmodule ExDoc.Formatter.HTML.Autolink do
           "[#{text}](#{elixir_docs}Kernel.SpecialForms" <>
             "#{extension}##{prefix}#{enc_h(function)}/#{arity})"
 
+        module in modules_refs && module_id ->
+          IO.warn("#{match} is not found (parsing #{module_id} docs)", [])
```

Comment:
Can we also show the function name/arity in case the docs come from those? Or we don't have this info?

Reply:
not sure what you mean, could you elaborate?

`match` is e.g. `c:CustomProtocol.plus_two/1`

Reply:
ohh, that would be super helpful. I'll look into it.

Reply:
@josevalim Good call, it should be much more useful now!

The build should be fine now but we still get the warnings I mentioned in the first comment.